pinc/wordcheck_engine.inc

```diff
@@ -777,17 +777,6 @@ function save_word_list($path, $words)
     // needed for the array == array comparison below
     $words = array_values($words);
 
-    // see if list has changed before saving it
```

Comment:
Instead of taking this out and doing a load/save, let's add a new function to this file that uses `touch()` to update the good word list file. That does less data shuffling and creates a single function we can use where we want to reject suggestions. Perhaps call it simply `touch_project_good_words_file($projectid)`.

Reply:
great idea. I thought of it just after I submitted the last change realizing it is such a waste otherwise.